cts-java/src/test/java/org/ga4gh/cts/api/g2p/GenotypePhenotypeSearchIT.java

```diff
@@ -0,0 +1,264 @@
+public class GenotypePhenotypeSearchIT {
+
+    private static Client client = new Client(URLMAPPING.getInstance());
+
+    /**
+     *  error if no parameters
+     * @throws AvroRemoteException if there's an unanticipated error
+     */
+    @Test
+    public void ensureCatchNoParametersSearch() throws AvroRemoteException {
+        try {
+            SearchGenotypePhenotypeRequest request = SearchGenotypePhenotypeRequest.newBuilder().build();
+            SearchGenotypePhenotypeResponse response = client.genotypePhenotype.searchGenotypePhenotype(request);
+            fail("should have thrown error");
+        } catch ( Exception e) {
+            // no-op, expected
+        }
+    }
+
+    /**
+     * search for feature
+     * @throws AvroRemoteException
+     */
+    @Test
+    public void simpleFeatureSearch() throws AvroRemoteException {
+        SearchGenotypePhenotypeRequest request = SearchGenotypePhenotypeRequest.newBuilder().build();
+        request.setFeature("KIT *wild");
+        SearchGenotypePhenotypeResponse response = client.genotypePhenotype.searchGenotypePhenotype(request);
+        assertThat(response).isNotNull();
+        assertThat(response.getAssociations()).isNotNull() ;
+        assertThat(response.getAssociations().get(0)).isNotNull() ;
+    }
+
+
+    /**
+     * search for evidence
+     * @throws AvroRemoteException
+     */
+    @Test
+    public void simpleEvidenceSearch() throws AvroRemoteException {
+        SearchGenotypePhenotypeRequest request = SearchGenotypePhenotypeRequest.newBuilder().build();
+        request.setEvidence("imatinib");
+        SearchGenotypePhenotypeResponse response = client.genotypePhenotype.searchGenotypePhenotype(request);
+        assertThat(response).isNotNull();
+        assertThat(response.getAssociations()).isNotNull() ;
+        assertThat(response.getAssociations().get(0)).isNotNull() ;
+    }
+
+
+
+    /**
+     * search for Phenotype
+     * @throws AvroRemoteException
+     */
+    @Test
+    public void simplePhenotypeSearch() throws AvroRemoteException {
+        SearchGenotypePhenotypeRequest request = SearchGenotypePhenotypeRequest.newBuilder().build();
+        request.setPhenotype("GIST");
+        SearchGenotypePhenotypeResponse response = client.genotypePhenotype.searchGenotypePhenotype(request);
+        assertThat(response).isNotNull();
+        assertThat(response.getAssociations()).isNotNull() ;
+        assertThat(response.getAssociations().get(0)).isNotNull() ;
+    }
+
+
+    /**
+     * search for Phenotype
+     * @throws AvroRemoteException
+     */
+    @Test
+    public void simplePhenotypeSearchNoFind() throws AvroRemoteException {
+        SearchGenotypePhenotypeRequest request = SearchGenotypePhenotypeRequest.newBuilder().build();
+        request.setPhenotype("FooBar");
+        SearchGenotypePhenotypeResponse response = client.genotypePhenotype.searchGenotypePhenotype(request);
+        assertThat(response).isNotNull();
```

Comment:
Another null test of the response.  I won't mark any others I see below this point, but they should all be fixed.